File: 1630-count-odd-numbers-in-an-interval-range/count-odd-numbers-in-an-interval-range.java
class Solution {
    public int countOdds(int low, int high) {

        if(low%2==1 || high%2==1){
            return (high-low)/2+1;
        }
        return (high-low)/2;

    }
}
/*
 int o = 0;
        for (int i = low; i <= high; i++) {
            if (i % 2 != 0) {
                o++;
            }
        }
        return o;
*/

File: 1630-count-odd-numbers-in-an-interval-range/count-odd-numbers-in-an-interval-range.c
int countOdds(int low, int high){
       
       if(low%2==0 && high%2==0){
           return (high-low)/2;
       }
       return (high-low)/2+1;
}

/*

 int count=0;
        for(int num=low;num<=high;num++){
            if(num%2==1){
                count++;
            }
        }

        return count;
*/